orderfulfiller/orderfulfiller.go

```diff
@@ -67,10 +67,8 @@ func (r *OrderFulfiller) Run(ctx context.Context) {
 	if r.shouldRefundOrders {
 		go r.startOrderTimeoutWorker(ctx)
 	}
-	if r.shouldFillOrders {
-		go r.startOrderFillWorkers(ctx)
-		r.dispatchOrderFills(ctx)
-	}
+	go r.startOrderFillWorkers(ctx)
+	r.dispatchOrderFills(ctx)
```

Comment:
the OrderFulfiller currently relies on the order fill workers to update the order status from pending -> timed out. The orderfulfiller is generally kind of confusing, and I'd probably factor out timeout logic if there were time